css/style.css

```diff
@@ -18,3 +19,14 @@ button {
   font-size: 18px;
```

Comment:
not a good idea to use this as px, instead use rems, also, by default is better to let the browser the default value, but if you want to work with a default value, then you could set it as `1.125rem` which by default it will be `18px` assuming a default the browser font settings of `16px` which is standard.